apps/expo/tsconfig.json

```diff
@@ -1,11 +1,12 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "@acme/tsconfig/base.json",
```

Comment:
Hey @juliusmarminge, this base.json wasn't synced in the pull request. Do you still have it?